wpilibj/src/main/java/edu/wpi/first/wpilibj/livewindow/LiveWindow.java

```diff
@@ -136,7 +136,7 @@ public static synchronized void addActuator(String subsystem, String name, Senda
    * @param moduleType A string that defines the module name in the label for the value
    * @param channel    The channel number the device is plugged into (usually PWM)
    * @param component  The reference to the object being added
-   * @deprecated Use {@link edu.wpi.first.wpilibj.SensorBase#setName(String, int)} instead.
+   * @deprecated Use {@link edu.wpi.first.wpilibj.SensorUtil#setName(String, int)} instead.
```

Comment:
These aren't correct.  SendableBase is what to call now..

Reply:
Looks ok to me

Reply:
SensorUtil doesn't contain a setName() method and doesn't inherit from any classes. Therefore, SensorUtil is the wrong class to use here.